src/DssSpell.t.sol

```diff
@@ -1402,218 +1321,202 @@ contract DssSpellTest is DssSpellTestBase {
+
+        uint256 starGuardLength = starGuardValues.length;
+        for(uint256 i; i < starGuardLength; ++i) {
+            StarGuardLike starGuard = StarGuardLike(starGuardValues[i].starGuard);
+            // Plot the payload as the Pause Proxy (admin)
+            vm.startPrank(pauseProxy);
+            starGuard.plot(address(payload), address(payload).codehash);
+            vm.stopPrank();
+
+            // Should be executable now
+            assertTrue(starGuard.prob(), "StarGuard/prob-not-true");
+
+            // Expect the starGuard to emit its Exec event upon exec()
+            vm.expectEmit();
+            emit Executed();
+            vm.expectEmit(true, false, false, false, address(starGuard));
+            emit Exec(address(payload));
+            address executed = starGuard.exec();
+            assertEq(executed, address(payload), "StarGuard/exec-wrong-target");
+
+            // Still owner of subProxy, and no longer plotted
+            assertEq(SubProxyLike(starGuardValues[i].proxy).wards(address(starGuard)), 1, "StarGuard/subProxy-wards-changed");
+            assertFalse(starGuard.prob(), "StarGuard/spell-not-cleared");
         }
+    }
 
-        // Check locked token migration
-        uint256 oftAfterBalance = usds.balanceOf(USDS_OFT);
-        uint256 nttManagerAfterBalance = usds.balanceOf(NTT_MANAGER);
-        assertGe(
-            oftAfterBalance,
-            oftPreviousBalance + nttManagerPreviousBalance,
-            "TestError/MigrationStep1/oft-adapter-balance-not-increased"
-        );
-        assertEq(
-            nttManagerAfterBalance,
-            0,
-            "TestError/MigrationStep1/ntt-manager-balance-not-zero"
-        );
-
-        // Check rate limit settings
-        (,uint48 outWindow2,,uint256 outLimit2) = oft.outboundRateLimits(SOL_EID);
-        (,uint48  inWindow2,,uint256  inLimit2) = oft.inboundRateLimits(SOL_EID);
-        assertEq(outWindow2, 1 days,          "TestError/MigrationStep1/outWindow-rl-not-set");
-        assertEq(inWindow2, 1 days,           "TestError/MigrationStep1/inWindow-rl-not-set");
-        assertEq(outLimit2, 10_000_000 * WAD, "TestError/MigrationStep1/outLimit-rl-not-set");
-        assertEq(inLimit2, 10_000_000 * WAD,  "TestError/MigrationStep1/inLimit-rl-not-set");
+    function testStarGuardDrop() public {
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done(), "TestError/spell-not-done");
 
-        // OFT send works now
-        oft.send{value: msgFee.nativeFee}(sendParams, msgFee, payable(address(this)));
-        assertEq(
-            usds.balanceOf(address(this)),
-            usdsBalanceBeforeSend - sendParams.amountLD,
-            "TestError/MigrationStep1/oft-send-didnt-work"
-        );
+        address[3] memory starGuards = [
+            addr.addr("GROVE_STARGUARD"),
+            addr.addr("KEEL_STARGUARD"),
+            addr.addr("OBEX_STARGUARD")
```

Comment:
Shall we check the relative subproxies too? I don't think they were tested in the previous spells.